common/src/main/java/com/spotify/spydra/model/SpydraArgument.java

```diff
@@ -124,6 +124,7 @@ public void setJobArgs(List<String> jobArgs) {
     public Optional<Integer> max = Optional.empty();
     public Optional<Double> factor = Optional.empty();
     public Optional<Boolean> downscale = Optional.empty();
+    public Optional<Integer> downscaleTimeout = Optional.of(10);
```

Comment:
Preferably leave this `Optional.empty()` to make an explicit choice of interval and downscaleTimeout that don't conflict. `autoScaler.downscaleTimeout.orElseThrow` will catch it being empty.

Reply:
SGTM -- I was just afraid of making a breaking chanage.

Reply:
As this is all still experimental I'm fine with making breaking changes. Your reply prompted me to think about this again though. I've drafted up a commit that only requires [`downscale_timeout` to be set `if (downscale)`](https://github.com/karth295/spydra/compare/master...spotify:grace). What do you think of that?